cloudsmith_cli/cli/commands/entitlements.py

```diff
@@ -132,6 +132,17 @@ def print_entitlements(opts, data, page_info=None, show_list_info=True):
 
     rows = []
     for entitlement in sorted(data, key=itemgetter("name")):
+        ent_updated_at = (
+            entitlement["updated_at"].strftime("%Y/%m/%d, %H:%M:%S")
+            if entitlement["updated_at"]
+            else None
+        )
+        ent_created_at = (
+            entitlement["created_at"].strftime("%Y/%m/%d, %H:%M:%S")
```

Comment:
Is this the same format as before? Kinda feels like we should just do iso8601, but not worth breaking an interface for probably.

Reply:
Yeah it's different - shall I keep it to iso?